lib/site_template/README.txt

```diff
@@ -0,0 +1,23 @@
+===============================
+Welcome to your new Jekyll Site
+===============================
+
+This file generated by Jekyll will neither be included in the site built
+when you run `jekyll build` (or `jekyll serve`) nor will it be read by
+git if you plan to have your site set up as a git repository.
```

Comment:
hmm, i'm fine with it not being checked into git, but i'd like for it to be a markdown document, since that's basically the universal standard for readmes these days